src/map/scale.js

```diff
@@ -36,10 +36,26 @@ export const getDomain = (data, setMin, setMax) => {
   return domain;
 };
 
-export const getDynamicDomain = data => {
+const getMinAndMax = data => {
   const min = Math.floor(Math.min.apply(null, data) * 10) / 10;
   const max = Math.ceil(Math.max.apply(null, data) * 10) / 10;
-  return [min, max];
+  return { min, max };
+};
+
+export const getDynamicDomain = data => {
+  // Calculate the dynamic range:
```

Comment:
isn't this going to be harder to control - because the outcome is determined by the dataset and not the configuration? It just seems weird to override a configured setting with a different dataset vs having the configurations change the settings predictably.

Reply:
This is only for the situation where you've manually defined the scale type as `dynamic`, in which case you need to account for different columns of the same sheet as having different mins and maxes.

Reply:
Basically if the min is above 50%, then the scale should be all blue. If the max is below 50%, then it should be all red. But if one column has values on either side of 50%, then 50 needs to be the middle value.

Reply:
ah ok - makes sense